src/app/modules/test-execution-service/test.execution.service.ts

```diff
@@ -21,15 +21,13 @@ export class ExecutedCallTreeNode {
   Message?: string;
   Enter: string;
   Leave?: string;
-  PreVariables?: VariableValue[];
+  PreVariables?: any;
```

Comment:
I know this wasn't introduced by this change, but aren't class properties usually named to start with a lower-case letter?